puppet-lint.gemspec

```diff
@@ -1,20 +1,36 @@
 $LOAD_PATH.push(File.expand_path('../lib', __FILE__))
 require 'puppet-lint/version'
 
-Gem::Specification.new do |s|
-  s.name = 'puppet-lint'
-  s.version = PuppetLint::VERSION.dup
-  s.homepage = 'https://github.com/puppetlabs/puppet-lint/'
-  s.summary = 'Ensure your Puppet manifests conform with the Puppetlabs style guide'
-  s.description = 'Checks your Puppet manifests against the Puppetlabs
-  style guide and alerts you to any discrepancies.'
+Gem::Specification.new do |spec|
+  spec.name = 'puppet-lint'
+  spec.version = PuppetLint::VERSION.dup
+  spec.homepage = 'https://github.com/puppetlabs/puppet-lint/'
+  spec.summary = 'Ensure your Puppet manifests conform with the Puppetlabs style guide'
+  spec.description = <<-EOF
+    Checks your Puppet manifests against the Puppetlabs style guide and alerts you to any discrepancies.'
+  EOF
 
-  s.files = Dir['CHANGELOG.md', 'HISTORY.md', 'README.md', 'LICENSE', 'lib/**/*', 'bin/**/*', 'spec/**/*']
-  s.test_files = s.files.grep(%r{\Aspec/})
-  s.executables = s.files.grep(%r{\Abin/}) { |f| File.basename(f) }
-  s.require_paths = ['lib']
+  spec.files = Dir[
+    'README.md',
+    'LICENSE',
+    'lib/**/*',
+    'bin/**/*',
+    'spec/**/*',
+  ]
+  spec.test_files = Dir['spec/**/*']
```

Comment:
`test_files` is ignored nowadays and recent versions of Rubocop suggest to remove this.

Reply:
👍